tests/README.md

```diff
@@ -27,22 +27,24 @@ Run all test starting with ``test_array_`
 These are dirty notes for the quickest way to test mssql queries manually in the way the tests work.
 
 1. Create a new dir. E.g. `.local_docker_test`
-2. Copy the original docker-compose file from directory `.docker` and remove everything except the qgis and the mssql container.
-3. Copy the original Dockerfile as well. Leave it like it is...
+2. Copy the original docker-compose file from directory `.docker` and remove everything except the qgis and the mssql container (remove the postgis dependency in qgis as well).
 4. Copy the original `run-docker-tests.sh` and remove everything except:
```

Comment:
Even if MD seems to handle this enumerations in a smart way, i.e., fixing missing numbers, this would be:

3.

And check the others after this one as well.